tpch/tpcbench.py

```diff
@@ -186,8 +186,28 @@ def main(
 
     args = parser.parse_args()
 
+    if (args.qnum != -1 and args.query is not None):
+        print("Please specify either --qnum or --query, but not both")
+        exit(1)
+
+    queries = []
+    if (args.qnum != -1):
+        if args.qnum < 1 or args.qnum > 22:
+            print("Invalid query number. Please specify a number between 1 and 22.")
+            exit(1)
+        else:
+            queries.append((str(args.qnum), tpch_query(args.qnum)))
```

Comment:
explicitly mention TPCH in the id

```suggestion
            queries.append((f"TPCH-{args.qnum)}", tpch_query(args.qnum)))
```